File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        //ask for height
        height = get_int("Height: "); 
    }
    while (height < 1 || height > 8);
    for (int i = 1; i <= height; i++)
    {
        //print spaces
        for (int j = height - i; j > 0; j--)
        {
            printf(" ");
        }
        //print #
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        // New row
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

while True:
    x = get_int("Height: ")
    if x >= 1 and x <= 8:
        break

i = 1
j = x - 1
k = 1

while i <= x:
    print(f" " * j, end="")
    j -= 1
    print(f"#" * k, end="")
    k += 1
    print()
    i += 1







